### PR TITLE
Implements Rubocop

### PR DESCRIPTION
**Pull request by jonathanly**
## PR Summary:
- [x] Implements Rubocop in app
- [x] Fixes rubocop violations in code base

[Link](https://trello.com/c/zkdK2cix/13-as-a-developer-i-want-to-use-rubocop-to-maintain-code-consistency) to trello card.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+Layout/FirstParameterIndentation:
+  Enabled: false
+Layout/IndentationConsistency:
+  Enabled: true
+Layout/MultilineBlockLayout:
+  Exclude:
+    - "spec/**/*"
+Metrics/AbcSize:
+  Exclude:
+    - "app/controllers/sessions_controller.rb"
```
Comment (apalski) [inline .rubocop.yml]: You don't need to exclude this one as at the moment you don't have a session controller

File: .rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,85 @@
+AllCops:
+  Exclude:
+    - "bin/setup"
+    - "db/schema.rb"
+    - "Gemfile.lock"
+    - "spec/spec_helper.rb"
+EndOfLine:
+  Enabled: false
+Layout/AlignParameters:
+  EnforcedStyle: with_fixed_indentation
```
Comment (apalski) [inline .rubocop.yml]: Have you checked that you need this one? Maybe remove it and check.

File: .rubocop.yml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    - "config/**/*"
+    - "bin/*"
+    - "db/*"
+Metrics/LineLength:
+  Max: 80
+  Exclude:
+    - "config/**/*"
+    - "db/schema.rb"
+Layout/MultilineMethodCallIndentation:
+  EnforcedStyle: indented
```
Comment (apalski) [inline .rubocop.yml]: Same here, have you checked you need it?

File: .rubocop.yml (hunk 2 above)
Comment (jonathanly) [reply .rubocop.yml]: Yep, this cop is required (:

File: .rubocop.yml (hunk 1 above)
Comment (jonathanly) [reply .rubocop.yml]: Thanks for picking that one up!
Fixed in 8669724

File: .rubocop.yml (hunk 3 above)
Comment (jonathanly) [reply .rubocop.yml]: Thanks Annette, removed with 8669724

File: .rubocop.yml (hunk 2 above)
Comment (apalski) [reply .rubocop.yml]: Other than my couple of comments, LGTM!

**Comment by jonathanly** [comment]
Thanks for the review, merging into master now.
Let me know if you have any additional comments and I'll amend them later on.

**Pull request merged**